lib/cluster.js

```diff
@@ -694,8 +696,10 @@ function workerInit() {
     if (!this.isConnected()) {
       process.exit(0);
     } else {
-      send({ act: 'exitedAfterDisconnect' }, () => process.disconnect());
-      process.once('disconnect', () => process.exit(0));
+      const processDisconnect = () => process.disconnect();
```

Comment:
@sam-github did you intend to @mention me here? In any case, I don’t really have a strong opinion. I definitely don’t mind keeping simple cases of `some.thing(() => done())` the way they are.